lua/rest-nvim/utils.lua

```diff
@@ -84,7 +84,7 @@ function utils.read_file(path)
 end
 
 function utils.parse_http_time(time_str)
-    local pattern = "(%a+), (%d+) (%a+) (%d+) (%d+):(%d+):(%d+) GMT"
+    local pattern = "(%a+), (%d+)[\\s-](%a+)[\\s-](%d+) (%d+):(%d+):(%d+) GMT"
```

Comment:
`pattern` here is lua pattern, not regex, therefore `\s` won’t work. You can use `%s` instead.